lib/python/pcm_placeholder.py.in

```diff
@@ -15,4 +15,4 @@ that has to be configured by CMake
 
 """
 
-PCMSOLVER_PARSE_DIR='@PCMSOLVER_PARSE_DIR@'
+PCMSolver_PARSE_DIR='@PCMSolver_PARSE_DIR@'
```

Comment:
Yeah, it's a problem with no good sol'n yet. Just good to monitor the use cases.